python/tk_desktop2/websockets/encryption_handler.py

```diff
@@ -66,7 +66,7 @@ def encrypt(self, payload):
         :param str payload: String to encrypt.
         :returns: Encrypted string
         """
-        return self._fernet.encrypt(payload)
+        return six.ensure_str(self._fernet.encrypt(six.b(payload)))
```

Comment:
fernet expect bytes, but we expect a string.

Reply:
What is the difference exactly between `six.b` and `six.ensure_binary`? I'm reading the docs and I don't understand the difference.